### PR TITLE
Add node script for testing a display presentation on cci build

### PR DESCRIPTION
**Pull request by rodrigopavezi**

### CODE REVIEW

**Comment by rodrigopavezi** [comment]
[Implement Twitter E2E [5]](https://trello.com/c/QQwSU6LZ/4650-implement-twitter-e2e-5)

**Comment by rodrigopavezi** [comment]
[Slides Widget E2E [5]](https://trello.com/c/eTHOl3lw/5049-slides-widget-e2e-5)

File: test-display-runner.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
+global.log = console;
```
Comment (santiagonoguez) [inline test-display-runner.js]: is this file a test of the test ?

**Comment by tejohnso** [comment]
Aren't [this](https://github.com/Rise-Vision/rise-launcher-electron-e2e/blob/master/routes/install-and-upgrade.js#L43) and [this](https://github.com/Rise-Vision/rise-launcher-electron-e2e/blob/master/routes/install-and-upgrade.js#L71) now broken?

**Pull request merged**